src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,149 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { SortType } from '../types/SortType';
+
+const filterPeople = (
```

Comment:
it would be great to create separate js files in `utils` folder and place these functions there